src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/image-list/image-list.component.html

```diff
@@ -1,7 +1,10 @@
 <ul ngbNav
     #nav="ngbNav"
-    class="nav-tabs">
-  <li ngbNavItem>
+    class="nav-tabs"
+    stateId="image-list">
+  <li ngbNavItem
+      cdStatefulTab
+      stateId="issues">
```

Comment:
We can simplify this by moving the directive to the parent `ngbNav` and use the provided methods and  attributes from ng-bootstrap.:
```suggestion
    class="nav-tabs"
    cdStatefulTab="image-list">
  <li ngbNavItem="issues">
```
You can see the full change at https://github.com/tspmelo/ceph/commit/e70fa6d3f64902ee8b1d9e4f9dbb4e137f725c55

Reply:
> We can simplify this by moving the directive to the parent `ngbNav` and use the provided methods and attributes from ng-bootstrap.:

You're right, that's a really good improvement.